packages/adminrouter/buildinfo.json

```diff
@@ -9,7 +9,7 @@
       "adminrouter": {
           "kind": "git",
           "git": "https://github.com/dcos/adminrouter.git",
-          "ref": "5e23b06be6c061b394d2123424675c542072a6f2",
+          "ref": "44bb0b2d119154f9b4ea9f1c3af184c278bb39b5",
```

Comment:
Adminrouter has been bumped. Now this bump only includes 3 commits to AR: debugging-support attempt1 (accidentally reverted by unified-proxy), unified-proxy, and cosmos-endpoint.
Needs a rebase, but you can keep the 44bb SHA